cfpb/consumerfinance.gov
# Move TCCP results to new card list view

Migrating away from a table results view to a new card view. I cleaned up our CSS a bit and removed the defunct patterns. For lack of a better word, I called the card variant `m-card--tabular` as it shows tabular data inside a card pattern. The entire card is a link a la our [topic cards](https://cfpb.github.io/design-system/patterns/cards#topic-card) but the heading and card innards are all novel.

See https://github.local/Design-Development/Design-Thinking-and-User-Research/issues/254

## How to test this PR

1. `./frontend.sh`
1. Visit the [cards page](http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/) and you'll see a new layout.
1. `yarn cypress run --spec test/cypress/integration/consumer-tools/credit-cards/explore-cards.cy.js`

## Screenshots

| before | after |
|--------|-------|
| <img width="1194" alt="Screenshot 2024-04-02 at 12 17 39 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/1060248/06690375-eb47-47d6-8b9b-8e798d88af38"> | <img width="1194" alt="Screenshot 2024-04-02 at 12 16 03 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/1060248/0e0ac861-b0fe-40ef-84ae-f545c7edfee5"> |

#### Browser testing

Visually tested in the following supported browsers:

#### Accessibility


#### Other
